capstone/src/main/resources/static/profile.html

```diff
@@ -44,18 +46,16 @@ <h1>My Profile</h1>
                 <div id="description"></div>
             </div>
         </div>
-
-        <div class="row" id="no-profile">
-            <p>You have not build your profile yet, please submit a profile </p>
-        </div>
     </div>
     <div class="profile-form general-container" id="user">
         <form action="save-individual" method="POST" id="user-form">
-            <label>Are you an individual or a club/organization?</label>
-            <select  id="user-type-toggle" name="user-type" onchange="toggleForm('user-type-toggle');">
-                <option value="individual">Individual</option>
-                <option value="organization">Organization</option>
-            </select>
+            <div id="user-select">
+                <label>Are you an individual or a club/organization?</label>
```

Comment:
minor but I think you could just write "Individual or Club" just like you would only write "First Name"